hardware_interface/src/resource_manager.cpp

```diff
@@ -426,6 +434,7 @@ class ResourceStorage
     for (auto & interface : interfaces)
     {
       auto key = interface.get_full_name();
+      RCUTILS_LOG_INFO_NAMED("resource_manager", "Command interface: '%s'", key.c_str());
```

Comment:
Why are we logging command interfaces and not state interfaces above?

Reply:
oops, that was just some debug code. Deleting it.